lib/tag-suggestions/index.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import appState from '../flux/app-state';
+import { tracks } from '../analytics';
+
+const { search } = appState.actionCreators;
+const { recordEvent } = tracks;
+
+export class TagSuggestions extends Component {
+  static displayName = 'TagSuggestions';
+
+  static propTypes = {
+    query: PropTypes.string.isRequired,
```

Comment:
this shouldn't be needed here since we're not using it. it can be "swallowed up" in `filteredTags` which gets set in `connect`